### DISCUSSION TITLE
Version?

### DISCUSSION THREAD

**Discussion by Quessian-Phaedrus-Sternabite** [svsdval/video2midi#14] [Q&A]
What version of python is this? Also, what is "rame" on line 2091? I'm trying to run it but I'm getting errors. What is a config parser? And finally, in the end, what is the clock statement?

**Comment by svsdval** [comment]
>What version of python is this?

v2m works on python version from 2.7 and above. 

>I'm trying to run it but I'm getting errors

In first take look to wiki in it you can find installing manual for GNU/Linux,MacOs & Windows, if they not help, tell me about your OS, I'll try to figurate what to do.. 

>What is a config parser? 

About config parser is config parser built-in python. 

>Also, what is "rame" on line 2091?

I don't see any variable "rame" on line 2091.

>And finally, in the end, what is the clock statement?

hm, i don't understand what talk you about? try to explain more what you meaning?
